src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1058,6 +1059,20 @@ private AttributeSyntax ParseAttribute()
                     static (p, closeKind) => p.CurrentToken.Kind == closeKind,
                     expectedKind, closeKind);
             }
+
+            static bool immediatelyAbort(AttributeArgumentSyntax argument)
+            {
+                // We can be very thrown off by incomplete strings in an attribute argument (especially as the lexer
+                // will restart on the next line with the contents of the string then being interpreted as more
+                // arguments).  Bail out in this case to prevent going off the rails.
+                if (argument.expression is LiteralExpressionSyntax { Kind: SyntaxKind.StringLiteralExpression, Token: var literalToken } &&
```

Comment:
What about interpolated strings with newlines in them?

Reply:
sure.  i can extend to that case as well.